packages/rich-text/src/usePasteTextContent.ts

```diff
@@ -32,9 +32,17 @@ import customExport from './customExport';
  * Compare this with `useHandlePastedText` that handles pasting text while being
  * in text edit-mode.
  */
-function usePasteTextContent(insertElement, preset) {
+// @todo this really returns the inserted element and props is Partial<Element>, so is preset.
+function usePasteTextContent(
+  insertElement: (
+    type: string,
+    props: Record<string, unknown>,
+    asBackground?: boolean
+  ) => Record<string, unknown>,
```

Comment:
Not sure if it makes sense to create a local type `Element` or add a todo and leave it as is until the elements library package to get converted 🤔

Reply:
Edit: probable makes sense to wait for #12090